modules/rest-util/src/blaze/middleware/fhir/output.clj

```diff
@@ -1,8 +1,13 @@
 (ns blaze.middleware.fhir.output
-  "JSON/XML serialization middleware."
+  "FHIR Resource serialization middleware.
+
+  Currently supported formats:
+  * standard: JSON, XML;
+  * special: binary."
   (:require
    [blaze.anomaly :as ba]
    [blaze.fhir.spec :as fhir-spec]
+   [blaze.fhir.spec.type :as fhir-type]
```

Comment:
please use the alias `type` here as every other require uses. If you are unsure, just search in the whole project for `blaze.fhir.spec.type :as` and you will the the common alias